src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -146,8 +152,11 @@ protected override void OnNextCore(Block block)
 
                         timerTask = null;
                     }
-                    // Start timer if it is not started already.
-                    else timerTask = timerTask ?? Task.Delay(BatchIntervalMs, this.nodeLifetime.ApplicationStopping);
+                    else
```

Comment:
since this else is changed to have braces, I suggest changing the single line else on line 144 to also have braces so that the styles match

Reply:
braces were added there just to have a comment inside and not like that:

```
else
    //qwe qwe qwe
    doSomething();
```

and not because the code was on a single line

Reply:
I did wonder that, but still to have a single line else jars the eye when you have a if with a body and even more so when inconsistent in the same method

Reply:
We don't have such a policy that would say there should not be else branches on a single line. It is not even inconsistent. The subjective feeling of one is irrelevant.

Reply:
I personally don't like else without braces if the `if` has braces.
We don't need a policy for that its much more readable.